glossary/public-code-definition.md

```diff
@@ -8,6 +8,6 @@ Public code is both civic code (like policy or regulation) and computer source c
 
 Because public code serves the public interest, it should be: open, legible, accountable, accessible and sustainable.
 
-By developing public code independent from but implementable in a local context, as well as documenting the development process openly, it can provide a building block for others to re-implement in their local context, take as a starting point to continue development or as a basis for learning.
+Public code can be building blocks for others to re-use and re-implement in their local context, creating their public code. It can serve as a starting point to continue development or as a basis for learning. To enable that, the development of public code needs to be independent from, but implementable in, those contexts. The development process must also be openly documented.
```

Comment:
```suggestion
Public code can comprise building blocks for others to re-use and implement in their local context, creating new instances of public code. It can serve as a starting point to continue development or as a basis for learning. To enable that, public code development needs to be independent from, but implementable in different contexts. The development process must also be openly documented.
```